README.md

```diff
@@ -1,6 +1,6 @@
 # FBShipIt
 
-[![Build Status](https://travis-ci.org/facebook/fbshipit.svg?branch=master)](https://travis-ci.org/facebook/fbshipit)
+![CI](https://github.com/facebook/fbshipit/workflows/CI/badge.svg)
```

Comment:
This ends up having a useless hyperlink. How about this to make a clickthrough go to the Actions tab?

`[![CI](https://github.com/facebook/fbshipit/workflows/CI/badge.svg)](https://github.com/facebook/fbshipit/actions?query=branch%3Amaster)`